integration-tests/test_manpage.py

```diff
@@ -0,0 +1,45 @@
+import gzip
+import pytest
+
+
+@pytest.mark.parametrize(
+    "option",
+    [
+        "--checkin",
+        "--compliance",
+        "--conf",
+        "--disable-schedule",
+        "--display-name",
+        "--enable-schedule",
+        "--group",
+        "--keep-archive",
+        "--list-specs",
+        "--logging-file",
+        "--net-debug",
+        "--no-upload",
+        "--offline",
+        "output-dir",
+        "--output-file",
+        "--quiet",
+        "--register",
+        "--force-reregister",
+        "--retry",
+        "--show-results",
+        "--silent",
+        "--status",
+        "--test-connection",
+        "--unregister",
+        "--validate",
+        "--verbose",
+        "--version",
+        "--payload",
+        "--content-type",
+        "--diagnosis",
+    ],
+)
+def test_manpage(option):
+    """Test insights-client man page entries"""
+    file = "/usr/share/man/man8/insights-client.8.gz"
+    opened_file = gzip.open(file, "rb")
+    content = opened_file.read().decode()
```

Comment:
if you replace the binary mode (`b`) with text mode (`t`), then you can avoid the extra byte `decode()`ing